### PR TITLE
created rule for Univision.com

### PR DESCRIPTION
**Pull request by thisisparker**

### CODE REVIEW

**Comment by J0WI** [comment]
See also `*.uvnimg.com`

**Comment by thisisparker** [comment]
Added a working uvnimg subdomain

File: src/chrome/content/rules/Univision.com.xml [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+			- usprcauth.dev.y.univision.com
+			- api.vmh.dev.y.univision.com
+			- cdn1.uvnimg.com
+			- cdn2.uvnimg.com
+			- cdn3.uvnimg.com
+			- cdn4.uvnimg.com
+			- lmx1.uvnimg.com
+-->
+<ruleset name="Univision.com">
+	<target host="www.univision.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Univision.com.xml]: Please add a comment for `uvnimg.com` and a target for `univision.com` (without `www`).

File: src/chrome/content/rules/Univision.com.xml [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+	<target host="wires.univision.com" />
+	<target host="wkaq580.univision.com" />
+	<target host="wqba.univision.com" />
+	<target host="origin.www.univision.com" />
+	<target host="qa.x.univision.com" />
+	<target host="x963fm.univision.com" />
+	<target host="sports.dev.y.univision.com" />
+	<target host="fe.y.univision.com" />
+	<target host="yo1051.univision.com" />
+	<target host="cdn6.uvnimg.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Univision.com.xml]: `cdn[1-9]` are all working for me, but may require a custom test url.

**Comment by J0WI** [comment]
@thisisparker would you like to finish this?

**Comment by thisisparker** [comment]
@J0WI I actually don't know how to write the test URLs! That's what's held me back. I can look into that, and I'm sure I can figure it out, but until I do I'm blocked.

**Comment by J0WI** [comment]
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls
Feel free to ask me for any questions.

**Comment by zoracon** [comment]
Closing due to no revisions made. Can reopen if interest for revision is indicated

**Pull request closed without merging**